src/applications/ivc-champva/10-10D/helpers/utilities.js

```diff
@@ -0,0 +1,9 @@
+export const getFileSize = num => {
+  if (num > 999999) {
+    return `${(num / 1000000).toFixed(1)} MB`;
+  }
+  if (num > 999) {
+    return `${Math.floor(num / 1000)} KB`;
+  }
+  return `${num} B`;
+};
```

Comment:
Is there any reason we aren't using `else if` and do we not need a `toFixed()` for the KB?
```export const getFileSize = num => {
  if (num > 999999) {
    return `${(num / 1000000).toFixed(1)} MB`;
  } else if (num > 999) {
    return `${(num / 1000).toFixed(1)} KB`;
  } else {
    return `${num} B`;
  }
};

Reply:
I pulled it from elsewhere in the codebase, so not sure why it is the way that it is, but your suggestion is definitely a lot more uniform.